.travis.yml

```diff
@@ -7,28 +7,25 @@ addons:
       - python3
       - python3-yaml
 install: true
-script:
-- ./gradlew --no-daemon check
-after_success:
-- chmod +x ./.travis/update_checkstyle_thresholds && ./.travis/update_checkstyle_thresholds
+after_success: ./.travis/update_checkstyle_thresholds
 before_deploy:
 ## Run the travis setup, will download install4j and configure it
-- chmod +x ./.travis/setup && ./.travis/setup
+- ./.travis/setup
 ## Add the travis build number to the game engine version
 - GAME_CONFIG=game_engine.properties
 - ENGINE_VERSION=$(grep "engine_version" $GAME_CONFIG | sed 's/.*= *//g')
 - export TAGGED_VERSION="$ENGINE_VERSION.$TRAVIS_BUILD_NUMBER"
 - sed -i "s/engine_version.*/engine_version = $TAGGED_VERSION/" $GAME_CONFIG
 ## Run the gradle release build process - creates the installers
-- ./gradlew --no-daemon release
```

Comment:
Not sure about that... Is there any good reason we are not using the gradle daemon?